CMakeLists.txt

```diff
@@ -544,18 +544,20 @@ ELSE()
 ENDIF()
 
 IF (SECURITY_FOUND)
-  ADD_DEFINITIONS(-DGIT_SECURE_TRANSPORT)
```

Comment:
This commit does a bit more than its title states, as it also removes the definitions for platform-specific TLS specs. There are additional uses of these defines which you haven't covered in this commit, though:

- `GIT_SECURE_TRANSPORT` is used in stransport_stream.c and tls_stream.c
- `GIT_OPENSSL` is used in global.h, netops.h, openssl_stream.{c,h}